client/modules/User/pages/EmailVerificationView.jsx

```diff
@@ -81,7 +75,7 @@ EmailVerificationView.propTypes = {
     'checking',
     'verified',
     'invalid'
-  ]),
+  ]).isRequired,
```

Comment:
I believe that this will be `undefined` on the first render?  So it would fail the `propTypes` checks if we make it required.